### ISSUE TITLE
Duplicate Requests Being Displayed on Advanced Search

### ISSUE THREAD

**Issue by m-prodan** [bcgov/foi-flow#2094]
**Describe the bug in current situation**

**Link bug to the User Story**

**Impact of this bug**
High - will confuse users if multiple requests/results are pulled using the same ID number. 

**Chance of Occurring: Unknown - difficult to replicate**

**Pre Conditions: which Env, any pre-requesites or assumptions to execute steps?**
unknown - unable to replicate at this time, may need back-end analysis on specific requests.

**Steps to Reproduce**
1. Go to Advanced Search
2. Click on filter by request ID and enter LBR-2022-00015 or LBR-2022-12345
3. Scroll down to results
4. See multiple entries

![image.png](https://images.zenhubusercontent.com/6081a4c82f0c363dddb9c1a0/a3eaa54a-7d40-4a48-ae07-19ca50276744)

![image.png](https://images.zenhubusercontent.com/6081a4c82f0c363dddb9c1a0/e9eb95ab-5c2a-4d38-8d09-3add4a58c330)

**Actual/ observed behaviour/ results**

**Of Note** - LBR-2022-00015 - the request in the state of "Intake in Progress" does NOT appear in the team's queue. For LBR-2022-12345 - all of the duplicated requests appear in "my team's requests" when searched.

**Expected behaviour**
Requests with the same ID should not be present in the system or appearing on search.

**Comment by divyav-aot** [comment]
**LBR-2022-00015** - for this issue we suspect it is a multiple save issue. 
To fix this issue, we stopped allowing multiple save from the FE and also added another condition to check the duplicate AXIS ID in the BE. 
**LBR-2022-12345** - User selected multiple ministries while creating the request.
To fix this issue, we added a validation in the FE to allow the user to select only 1 ministry at a time. Also, added another check with the bcgovcode coming from AXIS with FOI bcgovcode to select the proper ministry.
![image.png](https://images.zenhubusercontent.com/6081a4273965c56c02fdc82c/412d6b23-9905-4c19-acbb-04e520ca7054)
